src/test/data/JsonRecipeBookStorageTest/invalidAndValidRecipeBook.json

```diff
@@ -0,0 +1,51 @@
+{
+  "_comment": "Invalid Steps field",
```

Comment:
This comment suggests that the following recipe should have an invalid steps field, however, it doesn't appear to be the case. Could this be an error?

Reply:
Thanks for the heads up Robin! It should be `Invalid Tags` instead.